docs/docs/user/examples/atm-fraud-pipeline.md

```diff
@@ -83,20 +85,21 @@ kubectl port-forward --namespace kpops service/k8kafka-cp-kafka-connect 8083:808
     --config ./examples/bakdata/atm-fraud-detection/config.yaml \
     --execute
     ```
-   
+
 !!! Note
-    You can use the `--dry-run` flag instead of the `--execute` flag and check the logs if your pipeline will be
```

Comment:
@raminqaf Do you know if this indent is important?

Reply:
The indent is important we should not delete it